### PR TITLE
Update Abcissa dev-dependency

### PR DESCRIPTION
**Pull request by hdevalence**
Follows @tarcieri's suggestion here: https://github.com/ZcashFoundation/zebra/pull/105#discussion_r299705959

### CODE REVIEW

**Review by tarcieri — APPROVED** [review]

**Comment by tarcieri** [comment]
@hdevalence you might try running `cargo check` and see if it impacts `Cargo.lock`, as it appears to be checked in

**Comment by hdevalence** [comment]
Hmm, running `cargo check` gives
```
error: failed to parse manifest at `/home/hdevalence/code/zebra/zebrad/Cargo.toml`

Caused by:
  invalid type: boolean `false`, expected a version string like "0.9.8" or a detailed dependency like { version = "0.9.8" } for key `dev-dependencies.default-features`
```

File: zebrad/Cargo.toml [hunk 1]
```diff
@@ -16,5 +16,7 @@ default-features = false
 features = ["application", "signals", "secrets", "time"]
 
 [dev-dependencies]
```
Comment (tarcieri) [inline zebrad/Cargo.toml]:
```suggestion
[dev-dependencies.abscissa]
```

Comment (hdevalence) [reply zebrad/Cargo.toml]: Oh, nice one, lol

**Review by dconnolly — APPROVED** [review]
LGTM, shall I merge @hdevalence ?

**Comment by hdevalence** [comment]
SGTM! We may want to update to a git version for a little while (to pick up dev changes) but this is good in the mean time.

**Comment by tarcieri** [comment]
@hdevalence I'll try to get 0.2 out by EOW, but would recommend switching to one of the git revisions for now if you'd like the bugfixes immediately

**Pull request merged**